mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -101,3 +118,112 @@ func ignitionFailure(c cluster.TestCluster) error {
 		return nil
 	}
 }
+
+// get current path and create tmp dir
+func GetTempdir(dirname string) (string, error) {
```

Comment:
I meant adding a new `GetTempdir()` in the builder code that returns the tempdir it already allocates.

Reply:
I noticed there's a [TempFile func](https://github.com/coreos/coreos-assembler/blob/main/mantle/platform/qemu.go#L550-L554) in the qemu.go already. Decided to use this - I hope that's ok.